r-pkg/LICENSE

```diff
@@ -1,3 +0,0 @@
-YEAR: 2017
-COPYRIGHT HOLDER: Uptake Technologies Inc.
-ORGANIZATION: Uptake Technologies Inc.
```

Comment:
I'm a bit confused as to what's happening with the LICENSE. Looks like we're keeping around the BSD-3 clause LICENSE.MD in the top level and deleting this text file? But its language is still around in the pkgdown LICENSE.html? Perhaps we should consider just the BSD-3 clause instead of 
![image](https://user-images.githubusercontent.com/16600275/32732491-5ec7ad14-c852-11e7-8bc9-3d6cf4c556fa.png)

Reply:
ugh what?? I didn't realize the two files were difference

yeah I messed this up. We probably need to keep both files

Reply:
We need to keep around both files. the language might have just made it into `pkgdown` because of the order I did things